packages/http/libraries/core/src/http/models/MetadataKey.ts

```diff
@@ -2,8 +2,11 @@ export const METADATA_KEY: {
   controller: string;
```

Comment:
Let's rely on multiple constants instead of in a single object. Consider `@inversifyjs/core` reflect keys as reference

Reply:
Fixed